htdocs/core/class/html.form.class.php

```diff
@@ -2961,6 +2962,11 @@ public function select_produits_list($selected = 0, $htmlname = 'productid', $fi
 		}
 
 		$sql .= " FROM ".$this->db->prefix()."product as p";
+
+		if (getDolGlobalString($conf->global->MAIN_SEARCH_PRODUCT_FORCE_INDEX)) {
```

Comment:
getDolGlobalStrin('MAIN_...')

Reply:
Thank you. I forgot to read the manual. :(